### PR TITLE
Fix README.md TF_VARs

### PR DESCRIPTION
**Pull request by saschagrunert**
Just to complete the required variables and remove the space in the `export` command.

### CODE REVIEW

**Comment by saschagrunert** [comment]
Ah, we probably should change the _**About**_ text of the project to mention 4.9 instead of 4.5.

**Comment by displague** [comment]
@saschagrunert have your run this using Route53?

File: README.md [hunk 1]
```diff
@@ -70,8 +70,8 @@ cd terraform-metal-openshift
 
     export TF_VAR_cluster_basedomain="domain.com"
     export TF_VAR_ocp_cluster_manager_token="eyJhbGc...d8Agva"
-    export TF_VAR_dns_provider = "cloudflare" # aws and linode are also supported
-    export TF_VAR_dns_options = {"api_token": "abc..."} # needed for aws and linode
+    export TF_VAR_dns_provider="cloudflare" # aws and linode are also supported
+    export TF_VAR_dns_options={"email": "abc@xyz.com", "api_key": "...", "api_token": "..."} # needed for aws and linode
```
Comment (displague) [inline README.md:74]:
```suggestion
    export TF_VAR_dns_options='{"email": "abc@xyz.com", "api_key": "...", "api_token": "..."}' # fields differ by DNS provider
```

**Comment by saschagrunert** [comment]
> @saschagrunert have your run this using Route53?

No, cloudflare.

**Pull request merged**